test/src/660-store_publish_statistics/main

```diff
@@ -182,56 +191,64 @@ cvmfs_run_test() {
     local files_added=$(cat test_660_$i | cut -d '|' -f 2)
     local files_removed=$(cat test_660_$i | cut -d '|' -f 3)
     local files_changed=$(cat test_660_$i | cut -d '|' -f 4)
-    local directories_added=$(cat test_660_$i | cut -d '|' -f 5)
-    local directories_removed=$(cat test_660_$i | cut -d '|' -f 6)
-    local directories_changed=$(cat test_660_$i | cut -d '|' -f 7)
-    local bytes_added=$(cat test_660_$i | cut -d '|' -f 8)
-    local bytes_removed=$(cat test_660_$i | cut -d '|' -f 9)
+    local duplicated_files=$(cat test_660_$i | cut -d '|' -f 5)
+    local directories_added=$(cat test_660_$i | cut -d '|' -f 6)
+    local directories_removed=$(cat test_660_$i | cut -d '|' -f 7)
+    local directories_changed=$(cat test_660_$i | cut -d '|' -f 8)
+    local bytes_added=$(cat test_660_$i | cut -d '|' -f 9)
+    local bytes_removed=$(cat test_660_$i | cut -d '|' -f 10)
+    local bytes_uploaded=$(cat test_660_$i | cut -d '|' -f 11)
 
     if [ $files_added != ${test_results[$i,1]} ]; then
```

Comment:
This check and all the following ones are giving me a warning on my machine:
```
src/660-store_publish_statistics/main: line 202: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 207: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 212: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 217: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 222: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 227: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 232: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 237: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 242: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 247: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 202: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 207: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 212: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 217: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 222: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 227: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 232: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 237: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 242: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 247: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 202: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 207: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 212: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 217: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 222: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 227: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 232: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 237: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 242: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 247: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 202: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 207: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 212: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 217: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 222: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 227: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 232: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 237: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 242: [: !=: unary operator expected
src/660-store_publish_statistics/main: line 247: [: !=: unary operator expected
```
I think these checks should be rewritten as follows:
```
if [ "x$something" != "x$other" ]; then
 ....
```
The problem is that when I fix the first warning the test no longer passes. Please see what happens in your case after you fix these checks (and the similar ones from test 659).

Reply:
On my machine (CentOS Linux release 7.5.1804) I don't get any warning.
If I modify as 
`if [ "x$something" != "x$other" ]; then`
 tests 659 and 660 still pass.